conf/messages

```diff
@@ -158,6 +158,7 @@ reassign-case.submit.button = Move back to queue
 suppress_case.form.state.required = Select yes if you have contacted the applicant
 reject_case.form.state.required = Select yes if you have contacted the applicant
 refer_case.form.state.required = Select yes if you have contacted the applicant
+remove_attachment.form.state.required = Select yes if you want to remove the attachment
```

Comment:
Minor possible usability/consistency issue. The error message suggests choose 'yes' to confirm, but the option is actually labelled 'Yes' (with capital Y). Quite pedantic, but should it align?
<img width="527" alt="Screenshot 2019-05-14 at 16 20 17" src="https://user-images.githubusercontent.com/20041831/57710273-7dbf3180-7664-11e9-946d-4e6417350a50.png">

Reply:
Noticed this is a common pattern throughout the application so probably not a problem.

Reply:
we don't normally capitalise words mid-sentence

Reply:
checked with Aimee and that is the required text